src/backend/joanie/core/api/client.py

```diff
@@ -1420,7 +1421,10 @@ class NestedOrderCourseViewSet(NestedGenericViewSet, mixins.ListModelMixin):
 
 
 class EventViewSet(
-    mixins.ListModelMixin, mixins.DestroyModelMixin, viewsets.GenericViewSet
+    mixins.CreateModelMixin,
+    mixins.ListModelMixin,
+    mixins.DestroyModelMixin,
```

Comment:
Do we really want to allow a user to delete an event ?

Reply:
Probably not.